README.md

```diff
@@ -54,11 +54,10 @@ positional arguments:
 optional arguments:
   -h, --help            show this help message and exit
   -a MANIFEST_ARGS, --manifest-args MANIFEST_ARGS
-                        A series of space separated key=value pairs. Each
-                        instance of $key in the manifest file is replaced by
-                        value verbatim. In particular, if you want to pass a
-                        string, it must be explicitly quoted, e.g.:
-                        ip="0.0.0.0"
+                        A list of key=value pairs. Each instance of '$<key>' in
+                        the manifest file is replaced by <value> verbatim. In
+                        particular, if you want to pass a string, it must be
+                        explicitly quoted, e.g.: 'ip="0.0.0.0"'
```

Comment:
We don't pass the entire manifest-args as a string anymore; instead we pass a list of strings so that we don't have to split on whitespace (which breaks when you try to use a manifest argument that has a space in it). I changed the comment to fit the new argument structure.